.swiftlint.yml

```
@@ -38,3 +38,4 @@ excluded:
  - DerivedData/
  - Derived/
  - Tuist/.build/
+ - Tuist/DerivedData
```

```suggestion
 - Tuist/DerivedData/
```
I works without it I'm sure, but it matches the rest of the file this way.